app/src/const.ts

```diff
@@ -1,6 +1,6 @@
 export const treemapPadding = 20
-export const bubblePadding = 35
-export const textSpacingFromCircle = 5
+export const bubblePadding = 10
+export const textSpacingFromCircle = 0
```

Comment:
Could this be removed then, if it is zero?

Reply:
Let's keep it for now. We might need an offset at some point